src/test/java/org/assertj/guava/api/RangeMapAssert_containsKeys_Test.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2014 the original author or authors.
+ */
+package org.assertj.guava.api;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.assertj.guava.api.Assertions.assertThat;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+import com.google.common.collect.TreeRangeMap;
+
+public class RangeMapAssert_containsKeys_Test extends RangeMapAssertBaseTest {
+
+  @Test
+  public void should_pass_if_actual_contains_given_keys() throws Exception {
+	// given
+
+	// when
+	Assertions.assertThat(actual).containsKeys(500, 600, 700);
+
+	// then
+  }
+
+  @Test
+  public void should_fail_if_actual_is_null() {
+	expectException(AssertionError.class, actualIsNull());
+	TreeRangeMap<Integer, String> actual = null;
+	assertThat(actual).containsKeys(500, 600, 700);
+  }
+
+  @Test
+  public void should_fail_if_keys_to_look_for_are_null() {
+	expectException(IllegalArgumentException.class, "The keys to look for should not be null");
+	assertThat(actual).containsKeys((Integer[]) null);
+  }
+
+  @Test
+  public void should_fail_if_keys_to_look_for_are_empty() {
+	expectException(IllegalArgumentException.class, "The keys to look for should not be empty");
+	assertThat(actual).containsKeys();
+  }
+
+  @Test
+  public void should_fail_if_actual_does_not_contain_all_given_keys() {
+	try {
+	  assertThat(actual).containsKeys(100, 200, 900);
+	} catch (AssertionError e) {
+	  assertThat(e).hasMessage("\n"
```

Comment:
I prefer to have line break for every `\n` and `+` at the end of the line so that it visually shows how the error message looks like. See [ShouldContainExactly_create_Test](https://github.com/joel-costigliola/assertj-core/blob/c15e3aca0bee8cfd1f251c78c38edfe9b15f4d60/src/test/java/org/assertj/core/error/ShouldContainExactly_create_Test.java) as an example.